cli/ballerina-cli/src/test/resources/test-resources/validJava11Project/main.bal

```diff
@@ -1,9 +1,9 @@
 import ballerina/jballerina.java;
 
 // Creates a file in the current directory.
-public function main(string filepath) {
+public function main(string filepath) returns error? {
    handle file = newFile(java:fromString(filepath));
-   boolean|error isSuccess = createNewFileInternal(file);
+   boolean _ = check createNewFileInternal(file);
```

Comment:
@pramodya1994, hope this and other project API test changes are OK.

Reply:
Changes looks ok to me.

Reply:
@MaryamZi Project API test resource changes look ok to me.